### ISSUE TITLE
Feature request: show equipped items

### ISSUE THREAD

**Issue by lvlvllvlvllvlvl** [Dav1dde/pasteofexile#7]
Hi, l assume this is something you've thought about, do you have any plans to implement it?

**Comment by Dav1dde** [comment]
Yes plans are there, I just don't know how to properly visualize gear yet.

**Comment by lvlvllvlvllvlvl** [comment]
You mean turning the item text into html that represents the gear? l thought there was a library that does that, though it's possible l was thinking about [poe-item-renderer](https://www.npmjs.com/package/poe-item-renderer), which hasn't been updated in 2 years and its source repo has been deleted. Would it be helpful if l found some current javascript project that does that and packaged it in a way that you could use it?

**Comment by Dav1dde** [comment]
There are multiple problems to this:

1) How to actually design the UI/HTML to display all that PoB exports (Note: What poe.ninja displays is just a subset. PoB has support for Item Sets and additional (unused) gear).
2) Depending how we solve 1), we need the assets. There is currently no easily accessible way to get base assets. There are a few JS libs around that can display gear but they also need to be supplied a link to the asset. The assets currently are only accessible through 2 ways (from PoE). Either through the trade river or through the game files. Hosted assets from the PoE CDN cannot be used because they are 'encrypted'.

I am myself not sure about 1) yet, for 2) I'll have to setup an asset pipeline to extract the assets from the game files and then host them somewhere. I don't think the river is an option for me here (iirc what poe.ninja does, but they already need the river for currency data etc.).

There is also potentially the option to use assets from poe.ninja or poedb, but that would probably require a new API from them and would need to be coordinated properly.

**Comment by lvlvllvlvllvlvl** [comment]
I've forked RePoE (the author hasn't updated it since September) and added some code to extract the images from the game files https://lvlvllvlvllvlvl.github.io/RePoE/ . Images are at the same path as the `dds_file` named in base_items.json and uniques.json, but with different extensions (both .png and .webp are available).
